samples/samples-powershell/AddProductParams/run.ps1

```diff
@@ -1,16 +1,18 @@
 using namespace System.Net
 
 # Trigger binding data passed in via param block
-param($Request)
+param($Request, $TriggerMetadata)
```

Comment:
Can you just remove $Request from the parameters? Since we're not even using them that would be best if that's allowed.